### ISSUE TITLE
Handle gifs

### ISSUE THREAD

**Issue by bunnymatic** [bunnymatic/slack_color_theme_generator#3]
I have a feeling that animated gifs (maybe all gifs) are not being properly handled.

Investigate and if they are not working, fix it.

This could be about the `imagemagick` install on the slackbot machine.  or... it might be that the histogram thing needs to do some preprocessing of gifs before it can get a histogram.

**Comment by bunnymatic** [comment]
On the deployed machine (a linode instance with imagemagick version like so:
```deploy@mau:~/deployed/slack_color_theme_generator$ convert -version
Version: ImageMagick 7.0.6-7 Q16 x86_64 2017-08-13 http://www.imagemagick.org
Copyright: © 1999-2017 ImageMagick Studio LLC
License: http://www.imagemagick.org/script/license.php
Features: Cipher DPC HDRI OpenMP
Delegates (built-in): bzlib djvu fontconfig freetype gvc jbig jng jpeg lcms lqr lzma openexr png tiff wmf x xml zlib
```

this image http://i.imgur.com/Ej9OGlX.gif

doesn't convert

``` 
deploy@mau:~/deployed/slack_color_theme_generator$ convert -colors 8 histogram:info:- -format "%c" /var/folders/h1/yt00n4r96bv52wc0g2_vh2jw0000gp/T//briefly-1502/briefly-991250-758661-1
convert: no decode delegate for this image format `INFO' @ error/constitute.c/ReadImage/509.
convert: no images defined `/var/folders/h1/yt00n4r96bv52wc0g2_vh2jw0000gp/T//briefly-1502/briefly-991250-758661-1' @ error/convert.c/ConvertImageCommand/3275.
```

This is probably also related to #1 because imagemagick fails but we don't get any response in slack or logs

**Comment by bunnymatic** [comment]
i'll look into this one.  probably as simple as updating imagemagick on the deployed box

**Comment by bunnymatic** [comment]
fixed with 4e04041
